canonicalwebteam/store_api/stores/snapstore.py

```diff
@@ -169,3 +169,14 @@ def update_store_snaps(self, session, store_id, snaps):
         )
 
         return self.process_response(response)
+
+    def update_store_invites(self, session, store_id, invites):
```

Comment:
I know it could make sense this method name, but it is doing exactly the same as `invite_store_members`

https://github.com/canonical-web-and-design/canonicalwebteam.store-api/blob/8a1118fd3b7203909c2488e8e124234041ded9e5/canonicalwebteam/store_api/stores/snapstore.py#L140-L149